cli/src/commands/pack.js

```diff
@@ -0,0 +1,81 @@
+const path = require('path')
+const { reporter, chalk, exit } = require('@dhis2/cli-helpers-engine')
+const fs = require('fs-extra')
+const makeBundle = require('../lib/makeBundle.js')
+const makePaths = require('../lib/paths.js')
+
+exports.command = 'pack [folder]'
+
+exports.describe = 'Create archive from the build.'
+
+exports.builder = yargs =>
+    yargs
+        .positional('folder', {
+            describe: 'The folder to pack relative to cwd.',
+            type: 'string',
+            defaultDescription: '.',
+        })
```

Comment:
`folder` seems a bit ambiguous here - could mean either input or output (and I would actually assume output on first glance).  Maybe `sourceDirectory` or `inputDirectory` and then the complementary `destinationDirectory` or `outputDirectory` or `targetDirectory`?

Reply:
OK, we use `source` `destination`/`dest` in other places so I'll align with that for consistency.